app/lib/sms-games/controllers/SGCompetitiveStoryController.js

```diff
@@ -102,7 +103,7 @@ SGCompetitiveStoryController.prototype.createGame = function(request, response)
       gameDoc.betas[idx] = {};
       gameDoc.betas[idx].invite_accepted = false;
       gameDoc.betas[idx].phone = phone;
-      gameDoc.betas[idx].name = request.body['beta_first_name_' + i] || null;
+      gameDoc.betas[idx].name = request.body['beta_first_name_' + i] || phone;
```

Comment:
Ahh great. I'm going to go ahead and merge this in, but make a note for myself to ask Marah & Alysha during our mobile checkin on Monday.